openlibrary/templates/covers/add.html

```diff
@@ -6,15 +6,19 @@
 $if doc.type.key == "/type/author":
     $ intro = _("There are two ways to place an author's image on Open Library")
     $ action = doc.url('/add-photo')
+    $ guideline = _("Image Guidelines")
 $elif doc.type.key == "/type/edition":
     $ intro = _("There are two ways to get a cover image on Open Library")
     $ action = doc.url('/add-cover')
+    $ guideline = _("Cover Guidelines")
 $elif doc.get_edition_covers:
     $ intro = _("There are three ways to get a cover image on Open Library")
     $ action = doc.url('/add-cover')
+    $ guideline = _("Cover Guidelines")
```

Comment:
```suggestion
```
The variables are set the same way in all cases except `$if doc.type.key == "/type/author":`.  This means that we can have one simple if statement — if the doc is an author then we can set the variables one way else we can set the variables another way.